packages/node/test/integration/take-action.spec.ts

```diff
@@ -0,0 +1,201 @@
+      new EventEmitter(),
+      new EventEmitter(),
+      memoryStoreService,
+      messagingService,
+      new InstructionExecutor(EMPTY_NETWORK),
+      EMPTY_NETWORK,
+      provider,
+      "node-A"
+    );
+
+    nodeB = new RequestHandler(
+      new ethers.utils.SigningKey(process.env.B_PRIVATE_KEY!).address,
+      new EventEmitter(),
+      new EventEmitter(),
+      memoryStoreService,
+      messagingService,
+      new InstructionExecutor(EMPTY_NETWORK),
+      EMPTY_NETWORK,
+      provider,
+      "node-B"
+    );
+
+    // use mocked store
+    nodeA.store = store;
+    nodeB.store = store;
+
+    const wallet = new ethers.Wallet(process.env.A_PRIVATE_KEY!, provider);
+    tttContract = await new ContractFactory(
+      TicTacToeApp.interface,
+      TicTacToeApp.bytecode,
+      wallet
+    ).deploy();
+  });
+
+  afterAll(() => {
+    firebaseServer.close();
+  });
+
+  it.skip("Node A and Node B having installed tic tac toe, Node A takes action, Node B acknowledges state update", async () => {
+    const appInstanceId = generateUUID();
+    const multisigAddress = getAddress(hexlify(randomBytes(20)));
+    const appInstance = createTTTAppInstance(
+      multisigAddress,
+      tttContract,
+      nodeA.selfAddress,
+      nodeB.selfAddress
+    );
+
+    when(
+      mockedStore.getMultisigAddressFromAppInstanceID(appInstanceId)
+    ).thenResolve(multisigAddress);
+    console.log(appInstanceId);
+    console.log(multisigAddress);
+
+    when(mockedStore.getStateChannel(multisigAddress)).thenResolve(
+      stateChannel
+    );
+
+    when(mockedStore.getChannelFromAppInstanceID(appInstanceId)).thenResolve(
+      stateChannel
+    );
+
+    when(
+      mockedStore.getAppInstanceIdentityHashFromAppInstanceId(appInstanceId)
+    ).thenResolve(appInstance.identityHash);
+
+    when(
+      mockedStateChannel.getAppInstance(appInstance.identityHash)
+    ).thenReturn(appInstance);
+
+    const takeActionRequest: NodeTypes.MethodRequest = {
+      params: {
+        appInstanceId,
+        action: {
+          actionType: 0,
+          playX: 0,
+          playY: 0,
+          winClaim: {
+            winClaimType: 0,
+            idx: 0
+          }
+        }
+      },
+      type: NodeTypes.MethodName.TAKE_ACTION,
+      requestId: generateUUID()
+    };
+
+    const takeActionResult = await nodeA.callMethod(
+      takeActionRequest.type,
+      takeActionRequest
+    );
+    console.log(takeActionResult);
+  });
+});
+
+function createTTTAppInstance(
+  multisigAddress: Address,
+  tttContract: Contract,
+  initiatorAddress: Address,
+  peerAddress: Address
+): AppInstance {
+  return new AppInstance(
+    multisigAddress,
+    [initiatorAddress, peerAddress],
+    0,
+    {
+      addr: tttContract.address,
+      applyAction: hexlify(randomBytes(4)),
+      resolve: hexlify(randomBytes(4)),
+      getTurnTaker: hexlify(randomBytes(4)),
+      isStateTerminal: hexlify(randomBytes(4)),
```

Comment:
For all intents and purposes you can just set these values as `"<not used, being removed>"`

Reply:
Will remove all together since https://github.com/counterfactual/monorepo/pulls?q=is%3Apr+is%3Aclosed